src/OpenyActivityFinderSolrBackend.php

```diff
@@ -164,10 +164,15 @@ public function runProgramSearch($parameters, $log_id) {
 
   /**
    * {@inheritdoc}
+   * @throws \Exception
    */
   public function doSearchRequest($parameters) {
     $index_id = $this->config->get('index') ? $this->config->get('index') : 'default';
     $index = Index::load($index_id);
+    if (!$index) {
+      $this->loggerChannel->error('Index not found.');
+      throw new \Exception('Index not found.');
```

Comment:
can we avoid throwing an error if we log a message?
Exceptions are great for developers, not for end users

Reply:
Just return with "nothing"

Reply:
@podarok thanks for the feedback. I've accepted the suggestion.